content/open-firmware-systems.md

```diff
@@ -36,27 +36,34 @@ The source code for our Open EC can be found [here](https://github.com/system76/
 ## Hardware utilizing Open Firmware/Open EC
 
 | System                       | Open Firmware | Open EC |
-|:----------------------------:|:-------------:|:-------:|
-| Adder WS (addw1)             | No            | No      |
-| Adder WS (addw2)             | Yes\*         | Yes\*   |
-| Adder WS (addw3 and newer)   | Yes           | Yes     |
-| Bonobo WS (bonw13 and older) | No            | No      |
-| Bonobo WS (bonw14 and newer) | Yes           | Yes     |
-| Darter Pro (darp5 and older) | No            | No      |
-| Darter Pro (darp6)           | Yes           | Yes\*   |
-| Darter Pro (darp7 and newer) | Yes           | Yes     |
-| Galago Pro (galp3 and older) | No            | No      |
-| Galago Pro (galp4)           | Yes           | No      |
-| Galago Pro (galp5 and newer) | Yes           | Yes     |
-| Gazelle (gaze14 and older)   | No            | No      |
-| Gazelle (gaze15)             | Yes\*         | Yes\*   |
-| Gazelle (gaze16 and newer)   | Yes           | Yes     |
-| Lemur Pro (lemp8 and older)  | No            | No      |
-| Lemur Pro (lemp9 and newer)  | Yes           | Yes     |
-| Meerkat (meer8 and older)    | No            | N/A     |
-| Meerkat (meer9)              | Yes           | N/A     |
-| Meerkat (meer10)             | No            | N/A     |
-| Oryx Pro (oryp5 and older)   | No            | No      |
-| Oryx Pro (oryp6 and newer)   | Yes           | Yes     |
+|:-----------------------------:|:-------------:|:-------:|
+| Adder WS (addw1)              | No            | No      |
+| Adder WS (addw2)              | Yes\*         | Yes\*   |
+| Adder WS (addw3 and addw4)    | Yes           | Yes     |
+| Adder WS (addw5)              | No\*\*        | No\*\*  |
+| Bonobo WS (bonw13 and older)  | No            | No      |
+| Bonobo WS (bonw14 and bonw15) | Yes           | Yes     |
+| Bonobo WS (bonw15-B and newer)| No\*\*        | No\*\*  |
+| Darter Pro (darp5 and older)  | No            | No      |
+| Darter Pro (darp6)            | Yes           | Yes\*   |
+| Darter Pro (darp7 and newer)  | Yes           | Yes     |
+| Galago Pro (galp3 and older)  | No            | No      |
+| Galago Pro (galp4)            | Yes           | No      |
+| Galago Pro (galp5 and newer)  | Yes           | Yes     |
+| Gazelle (gaze14 and older)    | No            | No      |
+| Gazelle (gaze15)              | Yes\*         | Yes\*   |
+| Gazelle (gaze16 and newer)    | Yes           | Yes     |
+| Lemur Pro (lemp8 and older)   | No            | No      |
+| Lemur Pro (lemp9 and newer)   | Yes           | Yes     |
+| Meerkat (meer8 and older)     | No            | N/A     |
+| Meerkat (meer9)               | Yes           | N/A     |
+| Meerkat (meer10)              | No            | N/A     |
+| Oryx Pro (oryp5 and older)    | No            | No      |
+| Oryx Pro (oryp6 to oryp12)    | Yes           | Yes     |
+| Oryx Pro (oryp13)             | No\*\*        | No\*\*  |
```

Comment:
oryp13 shouldn't have the TB asterisks. Since it's AMD, there are bigger challenges to solve than TB5 support.